### PR TITLE
[REDOX] Fix default keymap formatting

### PR DESCRIPTION
**Pull request by LongerHV**
This is just a visual alignment enhancement.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

**Review by fauxpark — APPROVED** [review]

File: lib/python/qmk/cli/json2c.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 """Generate a keymap.c from a configurator export.
 """
 import json
+import sys
```
Comment (fauxpark) [inline lib/python/qmk/cli/json2c.py:4]: Please undo these changes, or submit them in a separate PR.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: lib/python/qmk/cli/json2c.py (hunk 1 above)
Comment (LongerHV) [reply lib/python/qmk/cli/json2c.py]: Ofc, I intended these changes for a separate PR. My bad.

**Review by zvecr — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by fauxpark** [comment]
Thanks!
